docs/Roadmap.md

```diff
@@ -20,4 +20,20 @@ Any individual or organization can create an Issue on the OTP Roadmap using the
 ![choose-issue-type](images/roadmap-5.png)
 7. Fill in the necessary fields, and then click `Create`.
 <br>
-![create](images/roadmap-6.png)
\ No newline at end of file
+![create](images/roadmap-6.png)
+<be>
+
+
+A roadmap issue should follow these principles:
+
+Encourage discussions in the thread, [Gitter](https://app.gitter.im/#/room/#opentripplanner_OpenTripPlanner:gitter.im), PO- or developer meetings. Discussions should be concluded, and the roadmap description should be updated accordingly.
```

Comment:
Suggestion:
"Discussions about the Issue within the corresponding [Gitter](https://app.gitter.im/#/room/#opentripplanner_OpenTripPlanner:gitter.im) chat are encouraged along with participation in the product owner (PO) or developer meetings. Should these discussions reach a conclusion, the Roadmap Issue can be updated accordingly."